x/staking/types/msg.go

```diff
@@ -168,14 +168,14 @@ func (msg MsgEditValidator) ValidateBasic() sdk.Error {
 type MsgDelegate struct {
 	DelegatorAddr sdk.AccAddress `json:"delegator_addr"`
 	ValidatorAddr sdk.ValAddress `json:"validator_addr"`
-	Delegation    sdk.Coin       `json:"delegation"`
+	Amount        sdk.Coin       `json:"delegation"`
 }
 
-func NewMsgDelegate(delAddr sdk.AccAddress, valAddr sdk.ValAddress, delegation sdk.Coin) MsgDelegate {
+func NewMsgDelegate(delAddr sdk.AccAddress, valAddr sdk.ValAddress, amount sdk.Coin) MsgDelegate {
 	return MsgDelegate{
 		DelegatorAddr: delAddr,
 		ValidatorAddr: valAddr,
-		Delegation:    delegation,
+		Amount:        amount,
```

Comment:
`Amount`'s `Amount` field is accessed as `Amount.Amount` - bit repeatitive. How about changing `MsgDelegate.Amount` into `MsgDelegate.Value` or `MsgDelegate.Quid`[¹](https://www.latinitium.com/lewisshort?p=quid)?

Reply:
yeah, I'll change to `MsgDelegate.Value` - thanks @alessio